docs/src/main/sphinx/security/inspect-pem.rst

```diff
@@ -0,0 +1,137 @@
+.. _pem_certs:
+
+=========
+PEM files
+=========
+
+PEM (Privacy Enhanced Mail) is a standard for public key and certificate
+information, and an encoding standard used to transmit keys and certificates.
+The PEM format is specified in the :ref:`RFC 5958 <glossPEM>` standard.
+
+Trino supports PEM-encoded certificates without a container, and also supports
```

Comment:
I don't know what a "container" would refer to here.... these are all just ways to encode keys and certs.  I would just add a see also like the JKS docs have that say, if you wan to user JKS or PKCS 12 see the other docs.

Reply:
I found _container_ used as an umbrella term for PEM encoding and truststore encoding in several web resources. But yes, it's a heavily overloaded word.

Reply:
Also removed "container" throughout all docs in this PR.